Providers/Scripts/2.4x-2.5x/Scripts/Tests/run_test_OMS_nxProviders.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python
+#============================================================================
+# Copyright (c) Microsoft Corporation. All rights reserved. See license.txt for license information.
+#============================================================================
+import os
+import sys
+import glob
+
+orig_dir=os.path.realpath(os.getcwd())
+os.system('rm /tmp/omstest_cleanup* 2> /dev/null')
+os.system('rm -rf /tmp/omsext 2> /dev/null')
+
+if os.path.exists('/opt/micosoft/omsconfig'):
+    sys.stdout.write('Cannot run test_OMS_nxProviders.py when omsagent is present!') 
+    sys.exit(0)
+
+print 'Running test_OMS_nxProviders.py'
+
+# Extract the build
+os.system('touch /tmp/omstest_cleanup')
+bundle = sys.argv[1]
+bundle += '/omsagent*.universal.x64.sh' 
+bundle = glob.glob(bundle)[0]
+print 'Target = ' + bundle
+os.system('mkdir /tmp/omsext; cd /tmp/omsext ; ' + bundle + ' --extract')
+if os.system('which dpkg-deb &> /dev/null') == 0:
+    for pkg in glob.glob('/tmp/omsext/omsbundle*/100/*.deb'):
+        os.system('dpkg-deb -x ' + pkg + ' /tmp/omsconfig')
+else:
+    os.system('mkdir /tmp/omsconfig')
+    os.chdir('/tmp/omsconfig')
+    for pkg in glob.glob('/tmp/omsext/omsbundle*/100/*.rpm'):
+        os.system('rpm2cpio ' + pkg + ' | cpio -idmv')
+os.chdir('/tmp/omsconfig/opt/microsoft/omsconfig/module_packages')
+for pkg in glob.glob('./*.zip'):
+    os.system('unzip ' + pkg)
```

Comment:
zip and unzip are not installed by default. Can you update the PR description with all the dependent tools that need to be installed on build systems to run these tests?